test/s3select_test.cpp

```diff
@@ -2603,3 +2636,19 @@ TEST(TestS3selectFunctions, nested_query_multirow_result)
   s3select_res = run_s3select(input_query, input_csv);
   EXPECT_EQ(s3select_res, expected_res);
 }
+
+TEST(TestS3selectFunctions, opserialization_expressions)
+{
+  std::string input;
+  size_t size = 10;
+  generate_rand_columns_csv(input, size);
+
+  char a[5] = {'@', '#', '$', '%', '&'};
+  char x = a[rand() % 5];
+  char y = a[rand() % 5];
+
```

Comment:
the above may select the same character for a different purpose.
it better to remove the character from the string once it is selected.